src/plugins/litecoin/index.tsx

```diff
@@ -30,7 +30,7 @@ export default function register(): Plugins {
                 return new litecoin.ChainAdapter({
                   providers: { http, ws },
                   coinName: 'Litecoin',
-                  midgardUrl: getConfig().VITE_THORCHAIN_MIDGARD_URL,
+                  thorMidgardUrl: getConfig().VITE_THORCHAIN_MIDGARD_URL,
```

Comment:
same q here, curious why some plugins have both and some e.g this have only the THOR one?